File: java/src/main/java/gov/doc/ntia/sigmf/ext/annotation/environment/EmitterEnvironment.java
package gov.doc.ntia.sigmf.ext.annotation.environment;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class EmitterEnvironment extends Environment {

    private static final long serialVersionUID = 1L;

    @JsonProperty(value="ntia-environment:emitter_id", required = true)
    protected String emitter_id;

    public String getEmitter_id() {
        return emitter_id;
    }

    public void setEmitter_id(String emitter_id) {
        this.emitter_id = emitter_id;
    }


}

File: C#/sigmf/sigmf/ext/Annotationdir/environment/EmitterEnvironment.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace sigmf
{
    [Serializable()]
    public class EmitterEnvironment : Environment
    {
        private static readonly long serialVersionUID = 1L;

        [JsonProperty("ntia-environment:emitter_id")]
        protected string emitter_id;

        public string Emitter_id { get => emitter_id; set => emitter_id = value; }
    }
}
